webapp/src/DOMJudgeBundle/Resources/views/jury/partials/team_affiliation_form.html.twig

```diff
@@ -7,13 +7,44 @@
         {{ form_row(form.shortname) }}
         {{ form_row(form.name) }}
         <div class="form-group">
-            {{ form_label(form.country) }}
```

Comment:
Is it intentional that this label is dropped? Form looks a bit stranger now.

Reply:
Also I think we can now replace the whole `<div>` with just a `{{ form_row(form.country) }}` or even replace the whole form stuff with `{{ form(form) }}` since we do no custom stuff anymore

Reply:
Done!